src/components/test/Feed.test.tsx

```diff
@@ -91,5 +110,43 @@ describe("Feed", () => {
       button.simulate("click");
       expect(component.find(Post).length).toEqual(3);
     });
+
+    describe("Another User's Posts", () => {
+      it("Displays nav name in feed nav", async () => {
+        const component = mount(componentWithStore(Feed, store));
+        const postProfileBlock = component.findWhere((node) =>
+          node.hasClass("Post__metaBlock")
+        );
+        postProfileBlock.first().simulate("click");
```

Comment:
This is actually uneccessary since the change it makes in the store is already in the initial state. You should remove it to clarify clicking and store updates aren't actually under test. Once you remove it, the `waitFor` is uneccessary as well.